subsys/audio_modules/audio_module_template/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+menu "Audio Modules"
```

Comment:
How will this name work when several modules are added? Discuss.

Reply:
I would hope it would list all the avaliable modules, including the template, for the user to select which modules to include.